### DISCUSSION TITLE
Option: Use short format for date and time in lists

### DISCUSSION THREAD

**Discussion by flaviozluca** [Intermesh/groupoffice#836] [Q&A]
Friends,

What does this option do? I tried to enable/disable it and didn't see any difference in regards to the date/time in the email listing.

![image](https://user-images.githubusercontent.com/44004645/165791484-ae1aae28-327d-49ba-8eda-954635057c60.png)

Thanks.

**Comment by mschering** [comment]
When looking at the contact grid for example. Enable the Modified At column. If you have short format enabled you will see just the time if it was modified today. If it was yesterday you will see "Yesterday" or just the name of the day within this week. For older dates wou will see just the data. When you hover the mouse over it you will see the full date and time.

**Comment by flaviozluca** [comment]
Thanks, Merijn. Not for the email listing, though, right? Which makes it harder to comprenhend.

It seems a bit too specific. User doesn't know what the option does...

And I don't think it's documented, didn't find it in the manual...
